src/services/url.ts

```diff
@@ -22,9 +22,25 @@ class UrlService extends BaseService<UrlService.Options> {
   }
 
   init() {
+    this.beautifier = new UrlBeautifier(this.generateRoutes(), this.opts.beautifier, this.app.config);
```

Comment:
that way it can be available for other services to use when they `init()` if they need to